src/components/composites/Skeleton/SkeletonCircle.tsx

```diff
@@ -3,10 +3,10 @@ import { usePropsResolution } from '../../../hooks';
 import type { ISkeletonProps } from './types';
 import Skeleton from './Skeleton';
 
-const SkeletonCircle = ({ children, ...props }: ISkeletonProps) => {
+const SkeletonCircle = ({ children, ...props }: ISkeletonProps, ref: any) => {
   const resolvedProps = usePropsResolution('SkeletonCircle', props);
   // Skeleton component with border Radius 999
-  return resolvedProps.isLoaded ? children : <Skeleton {...resolvedProps} />;
+  return resolvedProps.isLoaded ? children : <Skeleton {...resolvedProps} ref={ref} />;
 };
 
 export default memo(forwardRef(SkeletonCircle));
```

Comment:
The fc is the same thing, just assigned elsewhere. It's props and then ref.

In other words the forward ref wants a function that takes props and a ref, which is what SkeletonCircle is. A function that accepts props and a ref. 

So I'm not sure what you mean. This seems correct to me.

Reply:
forwardRef technically is a typed function. The second type parameter being a props without the Ref.  This would've ensured typesafety, but because of how NativeBase used `any` in the lower level component the typecheck gets confused if you try to do it properly.

For now as long as the warning disappears I guess we should lave it at that.

Reply:
Forward ref (typescript, for that matter) doesn't care if you inline or pass through a variable.

But I'm not sure what you're suggesting. Do you mean the generics? Because while those are missing (consistently, which is not _good_), you can still type them in useRef. Not the prettiest, but also not the scope of this PR. It currently resolves (when typing in useRef) using no types at all.